about.html

```diff
@@ -32,36 +32,45 @@ <h1 class="section-header">About Speedometer 3</h1>
                                 <li>Measures the time to add, complete, and remove 100 todo items in a basic list.</li>
                                 <li>Each example implements the same todo application (TodoMVC) using different techniques and frameworks.</li>
                                 <li>
-                                    Workloads: <a href="resources/todomvc/vanilla-examples/javascript-es5/README.md" target="_blank">TodoMVC-JavaScript-ES5</a>,
-                                    <a href="resources/todomvc/vanilla-examples/javascript-es6-webpack-complex/README.md" target="_blank">TodoMVC-JavaScript-ES6-Webpack-Complex-DOM</a>,
-                                    <a href="resources/todomvc/vanilla-examples/javascript-web-components/README.md" target="_blank">TodoMVC-WebComponents</a>,
-                                    <a href="resources/todomvc/architecture-examples/react-complex/README.md" target="_blank">TodoMVC-React-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/react-redux/README.md" target="_blank">TodoMVC-React-Redux</a>,
-                                    <a href="resources/todomvc/architecture-examples/backbone/README.md" target="_blank">TodoMVC-Backbone</a>,
-                                    <a href="resources/todomvc/architecture-examples/angular-complex/README.md" target="_blank">TodoMVC-Angular-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/vue/README.md" target="_blank">TodoMVC-Vue</a>, <a href="resources/todomvc/architecture-examples/jquery/README.md" target="_blank">TodoMVC-jQuery</a>,
-                                    <a href="resources/todomvc/architecture-examples/preact-complex/README.md" target="_blank">TodoMVC-Preact-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/svelte-complex/README.md" target="_blank">TodoMVC-Svelte-Complex-DOM</a>,
-                                    <a href="resources/todomvc/architecture-examples/lit-complex/README.md" target="_blank">TodoMVC-Lit-Complex-DOM</a>
+                                    Workloads: <a href="https://github.com/WebKit/Speedometer/tree/release/3.0/resources/todomvc/vanilla-examples/javascript-es5" target="_blank">TodoMVC-JavaScript-ES5</a>,
```

Comment:
GitHub question: does this mean the branch will be named "release/3.0"?

Reply:
Yes.